Jenkinsfile.cd

```diff
@@ -94,6 +94,44 @@ def systemTests = { component ->
         sh "docker network prune -f"
     }
 
+	def dockerSetup = { testFileName ->
+		dir("environment/docker/pool") {
+            stage("Run pool ${testFileName}") {
+                 echo "$prefix: Running nodes..."
+                sh "./pool_start.sh $nodesNum"
+                pool_nodes = sh(script: 'docker ps', returnStdout: true)
+                echo "$prefix: $pool_nodes"
+            }
+
+            stage("Ensure node services started ${testFileName}") {
```

Comment:
Could you please surround test file name with parentheses or make the sentence more grammatically accurate? Please make the change for all similar cases (for `stage`).